src/dlgProfilePreferences.cpp

```diff
@@ -3200,17 +3215,12 @@ void dlgProfilePreferences::slot_setEncoding(const int newEncodingIndex)
     if (pHost) {
         pHost->mTelnet.setEncoding(comboBox_encoding->itemData(newEncodingIndex).toByteArray());
 
-        // When this was a tri-state checkbox setting we would store the
-        // partially checked setting (only, not the checked or unchecked values)
-        // into the Host class in order to cause the encoding to be checked so
-        // that it would determine what the (bool) Host::mWideAmbigousWidthGlyphs
-        // value should be.
-        // Since the control has been removed we now need to examine the value
-        // stored and if THAT equates to "Qt::PartiallyChecked" then do the same:
-        if (pHost->getWideAmbiguousEAsianGlyphsControlState()  == Qt::PartiallyChecked) {
-            // We are linking the Server encoding to this setting but we only
-            // need to refresh the setting for this if it is set to be automatic
-            // changed as necessary:
+        if (checkBox_useWideAmbiguousEastAsianGlyphs->checkState() == Qt::PartiallyChecked) {
+            // We are linking the Server encoding to this setting currently
```

Comment:
Same as previous comment - it is a better bet to keep it linked to the Server encoding after all!